tests/ctst/world/Zenko.ts

```diff
@@ -53,19 +61,19 @@ export default class Zenko extends World {
 
         if (this.parameters.AccountSessionToken) {
             CacheHelper.ARWWI[CacheHelper.AccountName] = {
-                AccessKeyId:  this.parameters.AccountAccessKey,
+                AccessKeyId: this.parameters.AccountAccessKey,
                 SecretAccessKey: this.parameters.AccountSecretKey,
                 SessionToken: this.parameters.AccountSessionToken,
-            }
-            this.cliMode.parameters.AssumedSession = CacheHelper.ARWWI[CacheHelper.AccountName];
-            this.cliMode.assumed = true;
+            };
+            CacheHelper.parameters.AccessKey = this.parameters.AccountAccessKey;
+            CacheHelper.parameters.SecretKey = this.parameters.AccountSecretKey;
+            CacheHelper.parameters.SessionToken = this.parameters.AccountSessionToken;
```

Comment:
Is it still used? I believe we removed the need to pass an ARWWI session to CTST but an account instead, as "AccountSessionToken" does not make sense, as accounts do not have any session tokens

Reply:
Yes this was because the branch wasn't rebased